tests/e2e/Employee.test.ts

```diff
@@ -0,0 +1,102 @@
+import { test, Page, expect } from "@playwright/test";
+
+let page: Page;
+
+test.beforeAll(async ({ browser }) => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  page = await browser.newPage();
+
+  // Click the Confirm button in the dialog that appears when the request is successful.
+  page.on("dialog", (dialog) => dialog.accept());
+
+  await page.goto("/employees");
+  await page.waitForLoadState("networkidle");
+});
+
+test.afterAll(async () => {
+  await page.close();
+});
+
+test.describe("Employee page", () => {
+  test("Reading employee", async () => {
+    expect(
+      await page.getByRole("table").getByRole("row").allTextContents()
+    ).toStrictEqual([
+      // table header
+      "IDName",
+      // table contents
+      "1foo",
+      "2bar",
+      "3baz",
+      "4error employee",
+    ]);
+  });
+
+  test("Creating employee", async () => {
+    await page.locator("text=/Create new employee/i").click();
```

Comment:
I think it's better to set `data-testid` instead get by text in case fail test case when you change text of button in future.

Reply:
I really don't like data-testid attribute because concerned about shipping that to production. (+ it is a good thing that the production code is changed and can be detected by tests.)

> PSA: before you use data-testid or container.querySelector, try using getByRole.
> https://twitter.com/kentcdodds/status/1157713671851614208?s=20&t=EX0iYNA8Y79Obt4FLyl2vA